web/src/app/schedules/ScheduleRouter.js

```diff
@@ -28,26 +28,24 @@ const query = gql`
     }
   }
 `
-class ScheduleList extends React.PureComponent {
-  render() {
-    return (
-      <SimpleListPage
-        query={query}
-        variables={{ input: { favoritesFirst: true } }}
-        mapDataNode={(n) => ({
-          title: n.name,
-          subText: n.description,
-          url: n.id,
-          isFavorite: n.isFavorite,
-        })}
-        createForm={<ScheduleCreateDialog />}
-        createLabel='Schedule'
-      />
-    )
-  }
+const ScheduleList = () => {
+  return (
+    <SimpleListPage
+      query={query}
+      variables={{ input: { favoritesFirst: true } }}
+      mapDataNode={(n) => ({
+        title: n.name,
+        subText: n.description,
+        url: n.id,
+        isFavorite: n.isFavorite,
+      })}
+      createForm={<ScheduleCreateDialog />}
+      createLabel='Schedule'
+    />
+  )
 }
-
-export default function ScheduleRouter() {
+ 
+export const ScheduleRouter = () => {
```

Comment:
Let's leave these as function declarations instead of consts for consistency across the app